src/utilities/getIndicatorTableData.js

```diff
@@ -3,25 +3,37 @@
  * Sustainable Solutions (NZ) Ltd. 2019
  */
 
+// eslint-disable-next-line import/no-cycle
```

Comment:
lots of cycles introduced with this PR :(

Reply:
Yeah, it's a bit difficult to avoid them just due to the way the utilities are structure. Should just be two new ones (but I did add disable comments for some other existing ones), both of which should be fixed by refactoring the utilities to indicator class methods. There's quite a lot of existing cycles just due to the way the code is split which I think are going to require a bit of a directory structure rethink to fix.